bgpd/bgp_vty.c

```diff
@@ -74,6 +74,8 @@
 
 FRR_CFG_DEFAULT_BOOL(BGP_IMPORT_CHECK,
 	{ .val_bool = true, .match_profile = "datacenter", },
+	{ .val_bool = true, .match_profile = "traditional", },
+	{ .val_bool = false, .match_version = "<= 7.3", },
```

Comment:
this list is "first match", so you won't ever get `false` since we only have datacenter and traditional profiles :)

Also `<= 7.3` would break if someone loads a config saved on `7.3.1`

```
	{ .val_bool = false, .match_profile = "traditional", .match_version = "< 7.4", },
	{ .val_bool = true, },
```
should do the trick

Reply:
fixed